raiden/api/rest.py

```diff
@@ -216,6 +215,16 @@ def normalize_events_list(old_list):
             new_event['initiator'] = to_checksum_address(new_event['initiator'])
         if name == 'EventPaymentSentSuccess' or name == 'EventPaymentSentFailed':
             new_event['target'] = to_checksum_address(new_event['target'])
+
+        # the following two were introduced temporary and will be removed
```

Comment:
Can you find a generic way to do this? Like going through all members and see if they are a serializable type, otherwise convert them to stings?

Reply:
I'll look into that. I know it's ugly this way.